pkg/apis/core/v1alpha2/types.go

```diff
@@ -110,6 +112,23 @@ type GroupSpec struct {
 	IPBlock *secv1a1.IPBlock `json:"ipBlock,omitempty"`
 }
 
+// GroupPhase defines the phase in which a Group is.
+type GroupPhase string
+
+// These are the valid values for GroupPhase.
+const (
+	// GroupPending means the Group has been accepted by the system, but it has not been processed by Antrea.
+	GroupPending GroupPhase = "Pending"
+	// GroupRealized means the Group has realized all of its GroupMembers.
+	GroupRealized GroupPhase = "Realized"
```

Comment:
I feel "Realized" does not best fit Groups, as it just means the Group members are computed? But we want to use the same terms for all CRDs?

Reply:
i considered "Processed" and "Computed". Was not sure about either so stick with what is already used in NetworkPolicyStatus

Reply:
Or do you think this (computed) is like a condition? @tnqn

Reply:
I just took a look at K8s API convention to understand the difference between phase and condition, looks like they are for same purpose and phase is a deprecated usage:
> Some resources in the v1 API contain fields called phase, and associated message, reason, and other status fields. The pattern of using phase is deprecated. Newer API types should use conditions instead. Phase was essentially a state-machine enumeration field, that contradicted system-design principles and hampered evolution, since adding new enum values breaks backward compatibility. 

https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#typical-status-properties

Reply:
thanks for the link.. makes sense to transition to Conditions.. here i propose the following:
```
status:
  conditions:
    - type: GroupMembersReady
      status: "True"/"False"
      lastTransitionTime: "2021-1-25T16:29:31Z"
```

Reply:
How about GroupMembersComputed?

Reply:
> How about GroupMembersComputed?

i don't have a strong opinion. I was now thinking of using the plain commonly used term `Ready` or `Succeeded` for `type` value, will post patch with GroupMembersComputed

Reply:
Ok. No strong preference either. Would let you decide.

Reply:
done